src/request/transaction.ts

```diff
@@ -117,12 +117,16 @@ export default class Transaction {
   private sanitizeKeys() {
     const sanitized = Object.assign({}, this) as any;
 
-    if (
-      sanitized.creditCard != null &&
-      sanitized.creditCard.last4digits != null
-    ) {
-      sanitized.creditCard.last_4_digits = this.creditCard!.last4digits;
-      delete sanitized.creditCard.last4digits;
+    if (sanitized.creditCard != null) {
+      if (sanitized.creditCard.last4digits != null) {
```

Comment:
Do we need to worry about `last4digits` here? Aren't we operating on a `CreditCard` object from above?

Reply:
Ah yeah. I was thinking we were operating on something implementing the `CreditCardProps` interface such that users may already have been supplying a class other than the provided `CreditCard`